.envrc

```diff
@@ -0,0 +1 @@
+use nix
```

Comment:
Should do
```
if has nix; then
  use nix
fi
```
to not spam non-nix users.

Reply:
Updated. I stole this from sdk-codegen. Let me fix there as well